runtime/exporter-prometheus/src/main/java/io/aklivity/zilla/runtime/exporter/prometheus/internal/printer/PrometheusMetricsPrinter.java

```diff
@@ -86,7 +87,7 @@ private String formatScalar(
             "# TYPE %s %s\n" +
             "%s{namespace=\"%s\",binding=\"%s\"} %f";
```

Comment:
```suggestion
            "%s{namespace=\"%s\",binding=\"%s\"} %s";
```

If we use `%s` formatter specifier instead, then `double` will box to `Double` and call `Double.toString()` which is locale-independent, so no need to specify locale variant of `String.format(...)`, agree?

Reply:
Yes, agreed! Thanks!